### PR TITLE
Update http -> mysql sample to adapt to app-starter design

### PR DESCRIPTION
**Pull request by sabbyanandan**
- includes steps to patch `jdbc` app with the driver
- removed `--include` property from stream definition

This sample will be updated once we have spring-cloud/spring-cloud-stream-app-starters#36 changes incorporated. 

Resolves spring-cloud/spring-cloud-dataflow-samples#7

### CODE REVIEW

File: streaming/http-to-mysql/README.adoc [hunk 1]
```diff
@@ -63,12 +63,48 @@ Welcome to the Spring Cloud Data Flow shell. For assistance hit TAB or type "hel
...
+- cd `jdbc-sink-kafka`
+- modify `pom.xml` to include `mysql` driver (_remove `hsqldb`_)
++
+[source,xml]
+----
+      <dependency>
+        <groupId>org.hsqldb</groupId>
+        <artifactId>hsqldb</artifactId>
+        <version>2.3.3</version>
+        <scope>exclude</scope>
```
Comment (artembilan) [inline streaming/http-to-mysql/README.adoc]: There is no such a `scope` in [Maven|https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html].
How about *remove* this `hsqldb` dependency or comment out in the XML style?

(I can do that on merge if only a concern)

File: streaming/http-to-mysql/README.adoc [hunk 2]
```diff
@@ -81,14 +117,14 @@ NOTE: If MySQL isn't running on default port on `localhost` or if you need usern
 dataflow:>stream list
 ```
 +
-. Notice that `mysqlstream-http` and `mysqlstream-jdbc` https://github.com/spring-cloud/spring-cloud-stream-modules/[Spring Cloud Stream] modules are running as Spring Boot applications within the `server` as a collocated process.
+. Notice that `mysqlstream-http` and `mysqlstream-jdbc` https://github.com/spring-cloud/spring-cloud-stream-modules/[Spring Cloud Stream] modules are running as Spring Boot applications within the Local `server` as collocated process.
```
Comment (artembilan) [inline streaming/http-to-mysql/README.adoc]: collocated process**es** ?

File: streaming/http-to-mysql/README.adoc [hunk 3]
```diff
@@ -197,6 +233,37 @@ Successfully targeted http://dataflow-server.app.io
...
+- cd `jdbc-sink-rabbit`
+- modify `pom.xml` to include `mysql` driver (_remove `hsqldb`_)
++
+[source,xml]
+----
+      <dependency>
+        <groupId>org.hsqldb</groupId>
+        <artifactId>hsqldb</artifactId>
+        <version>2.3.3</version>
+        <scope>exclude</scope>
```
Comment (artembilan) [inline streaming/http-to-mysql/README.adoc]: DITO.

**Comment by artembilan** [comment]
That's all. 
Just let me know and I'll fix on merge.

File: streaming/http-to-mysql/README.adoc (hunk 1 above)
Comment (sabbyanandan) [inline streaming/http-to-mysql/README.adoc]: Having `hsqldb` along with `mysql` driver conflicts. My bad, I meant to have it in test scope, so it isn't included in the final bits. I will push a commit.

**Comment by artembilan** [comment]
Merged as https://github.com/spring-cloud/spring-cloud-dataflow-samples/commit/e8ff0b448ec94d91066b8243e1f69d218d3e10cf

**Pull request closed without merging**